airbyte-integrations/connectors/source-salesforce-singer/source_salesforce_singer/source.py

```diff
@@ -74,20 +74,19 @@ def check_config(self, logger: AirbyteLogger, config_path: str, config: json) ->
     def discover_cmd(self, logger, config_path) -> str:
```

Comment:
did you have a chance to check `BaseSingerSource` class, it simplifies some of this routine, do you much benefit from using it here?